source/Public/Connect-FabricAccount.ps1

```diff
@@ -35,12 +38,16 @@ function Connect-FabricAccount {
 
     Connects as Service Principal using id and secret
 
+.OUTPUTS
+    None. This function does not return any output.
+
 .NOTES
 
-    Revsion History:
+    Revision History:
```

Comment:
thoughts on just removing revision history from commands? I think we have that info in GitHub anyway?

cc @Jojobit, @SQLDBAWithABeard - opinions?

Reply:
@jpomfret I was thinking the same, but let's create an issue for that.